product_docs/docs/pem/9/supported_platforms.mdx

```diff
@@ -9,3 +9,13 @@ For information about the platforms and versions supported by PEM, see [Platform
     Postgres Enterprise Manager 8.3 and later is supported on SLES.
 
 ## Database compatibility
+
+The following table provides information about the PEM versions and their supported corresponding versions of PostgreSQL (PG), EDB Postgres Extended Server (PGE), EDB Postgres Advanced Server (EPAS), and EDB Postgres Distributed (PGD).
+
+|           |**PEM 8.x**                   |                    |**PEM 9.x**                    |                         |
```

Comment:
I think we don't need PEM 8.x here as it is already added in PEM 8 docs. Same comments as above for header and reordering the database server names.

Reply:
Done